doc/api/http2.md

```diff
@@ -271,12 +271,17 @@ server.on('stream', (stream, headers) => {
     'content-type': 'text/html',
     ':status': 200
   });
+  stream.on('error', (error) => console.error(error));
```

Comment:
Nit: how about passing `console.error` directly? It's already bound. If this is the prevalent usage in the docs, ignore this comment.